src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -61,7 +61,16 @@ int CondTraverseToString(const OpBase *ctx, char *buff, uint buff_len) {
 	offset += snprintf(buff + offset, buff_len - offset, "%s | ", op->op.name);
 	offset += QGNode_ToString(op->ae->src_node, buff + offset, buff_len - offset);
 	if(op->ae->edge) {
```

Comment:
What if the no edge is referenced, e.g.
`MATCH (a)-[]->(b)<-[]-(c) RETURN a,b,c`

Reply:
Not ideally, that (and all other formations that contain no edge references) print as:
`Conditional Traverse | (a)->(b)"`

This could definitely be improved, but seems out of scope for this PR.

Reply:
👍